kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/DerivedResourcesTest.java

```diff
@@ -193,22 +191,9 @@ Stream<DynamicContainer> dependentResourcesShouldEqual() {
         return dependentResourcesShouldEqual(list);
     }
 
-    static List<Path> filesInDir(Path dir, Pattern pattern) {
-        var result = new ArrayList<Path>();
-        try (var expected = Files.newDirectoryStream(dir, path -> pattern.matcher(path.getFileName().toString()).matches())) {
-            for (Path f : expected) {
-                result.add(f);
-            }
-        }
-        catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
-        return result;
-    }
-
     Stream<DynamicContainer> dependentResourcesShouldEqual(List<DesiredFn<KafkaProxy, ?>> list) {
```

Comment:
Not for this PR, but I think we should drop the `@TestFactory` from DRT and just have `@Test` methods for each directory. That will make debugging a _lot_ easier.